position_controller:
  type: position_controllers/JointPositionController
  joint: pan_joint

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
  